.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Describe the bug**
```

Comment:
After finding the "rich diff" button: I think there needs to be a second line break after each `**heading**`. You could also consider using `### heading` (or some other level) instead.